webpack/components/AnsibleRoles/components/AnsibleRolesImportButton.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { importUrl } from '../AnsibleRolesPageHelpers';
+
+export const AnsibleRolesImportButton = () => {
+  let dropdownItems;
+
+  const {
+    response: { results },
+    status,
+  } = useAPI('get', '/api/v2/smart_proxies');
+
+  const [isOpen, setIsOpen] = React.useState(false);
+  const onToggle = isToggleOpen => {
+    setIsOpen(isToggleOpen);
+  };
+  const onFocus = () => {
+    const element = document.getElementById('toggle-primary');
+    element.focus();
+  };
+  const onSelect = () => {
+    setIsOpen(false);
+    onFocus();
+  };
+
+  if (status !== 'RESOLVED') {
+    dropdownItems = [
+      <DropdownItem key="loading_item">{__('Loading...')}</DropdownItem>,
+    ];
+  } else {
+    dropdownItems = results
+      .filter(result =>
+        result.features.some(feature =>
+          feature.capabilities.includes('ansible')
+        )
+      )
```

Comment:
This filters Smart Proxies with the Ansible feature on the client side. Can we please filter server side? The API should support `feature=Ansible`.

Reply:
Unfortunately, I couldn't get this to work...